jep/202/README.adoc

```diff
@@ -186,6 +196,26 @@ This temporary role creation does not exist in all clouds nor other artifact rep
 
 We opted for the first, simpler option.
 
+== Backwards Compatibility
+
+Existing plugins using `ArtifactManager` API will continue to work using the new selected implementation.
+
+However:
+
+* Various plugins call `Run.getArtifactsDir` and similar deprecated APIs.
+These would already have been broken for users of the Compress Artifacts plugin, but that is rarely used, whereas we are proposing lots of people run with the S3 artifact manager.
+Calls to the deprecated APIs will behave as if there were no artifacts in the build.
+We could add telemetry so that such calls produce a warning in the system log, at least when the build actually does have a custom artifact manager selected.
+
+* Some plugins using `VirtualFile` may still be calling open and then passing the stream to an agent.
+This will work, but will be very expensive when using S3 storage. They need to be updated to call `VirtualFile.toExternalURL`.
+
+== Security
+
+Security considerations make agents need to be restricted to only access the artifacts needed.
+Having access to the blob store would mean access to other jobs artifacts.
+
+Agents only do URL based upload/download operations and get the correct url to do so from the master.
 Extra care needs to be taken so agents do not have any access to the blob store.
```

Comment:
Can delete this sentence now? Or shift to the next paragraph?

Reply:
+1